tests/kernel/wmp_simple.pm

```diff
@@ -38,7 +42,11 @@ sub run {
     assert_script_run("systemctl set-property SAP.slice MemoryLow=$cgroup_mem");
 
     # start hana again and wait for the memory consumption to settle
-    assert_script_run('sudo -u ndbadm bash -c "/usr/sap/NDB/HDB00/exe/sapcontrol -nr 00 -function StartSystem ALL"');
+    my $admuser = lc($sid) . "adm";
```

Comment:
We have a function in `lib/sles4sap` that does this (`set_sap_info()`), but I understand not wanting to load that full library just for that. :)

Reply:
I was not aware of this - but I agree, pulling the full library might be a bit much. I don't have a strong opinion here.